modules/nf-core/jasminesv/main.nf

```diff
@@ -8,14 +8,14 @@ process JASMINESV {
         'biocontainers/jasminesv:1.1.5--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(vcfs), path(bams), path(sample_dists)
+    tuple val(meta), path(vcfs, arity:'1..*'), path(bams), path(sample_dists)
```

Comment:
this should be added into meta.yml 🗡️  
rest seems LGTM

Reply:
Is there an option for `arity` in the meta.yml?

Reply:
I'll merge this for now, I can still open another PR if things have been clarified :)